Jenkinsfile

```diff
@@ -40,6 +40,7 @@ pipeline {
       agent { label 'master || immutable' }
       options { skipDefaultCheckout() }
       steps {
+        sh 'env | sort'
```

Comment:
This is only for testing purposes as I wanna be sure what's the env variable at runtime. I'll remove it later on

Reply:
```suggestion
```